.github/workflows/release.yml

```diff
@@ -410,16 +410,6 @@ jobs:
           rm -v release/._ArduinoCreateAgent*.dmg
           mv -v ArduinoCreateAgent-windows*/* release/
 
-      - name: VirusTotal Scan
-        uses: crazy-max/ghaction-virustotal@v2
-        with:
-          vt_api_key: ${{ secrets.VIRUSTOTAL_API_KEY }}
-          vt_monitor: true
-          files: |
-            release/*.exe
-            arduino-create-agent-windows-2019-386/arduino-create-agent.exe
-            arduino-create-agent-windows-2019-amd64/arduino-create-agent.exe
-
```

Comment:
Removed in the release workflow because the workflow is not triggered by a release and so `update_release_body: true` cannot be used (not tested yet)